### PR TITLE
Adding secml

### PR DESCRIPTION
**Pull request by Walter-0**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-rsclient/bld.bat [hunk 1]
```diff
@@ -1,2 +1,2 @@
-"%R%" CMD INSTALL --build . %R_ARGS%
-IF %ERRORLEVEL% NEQ 0 exit /B 1
+"%R%" CMD INSTALL --build . %R_ARGS%
```
Comment (dopplershift) [inline recipes/r-rsclient/bld.bat:1]: Please don't modify any files outside your recipe.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Recipe is in good shape, just a minor tweak needed. Can someone from @mxr-conda confirm they are willing to be listed as a maintainer for this recipe?

File: recipes/r-rsclient/bld.bat (hunk 1 above)
Comment (Walter-0) [reply recipes/r-rsclient/bld.bat]: I reverted it. It was changed when I pulled the latest master and I was unable to unmodify it using git checkout --. git stash did not work either.

**Comment by mxr-conda** [comment]
@dopplershift Agree to maintain

**Comment by Walter-0** [comment]
@conda-forge I'm not sure how to deal with the merge conflict. I'm unable to stash or checkout that file and resolve the merge conflict on my end.

**Comment by goanpeca** [comment]
Hi @Walter-0 

Have you tried checking the latest version of staged recipes?

Something like (assumint a remote upstream is pointing to conda-forge/staged-recipes)

```
git checkout master
git pull upstream master
git checkout secml
git merge master
```

**Comment by Walter-0** [comment]
I deleted my fork and re-forked to fix that weird issue, now closing to open a new PR.
https://github.com/conda-forge/staged-recipes/pull/15936

**Pull request closed without merging**